h2o-py/h2o/model/PermutationFeatureImportance.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Permutation Feature Importance.
```

Comment:
Which file is used? PermutationFeatureImportance.py  or permutation_varimp.py? Keep only one, please.